docs/modules/monitoring/con-operator-scrape-metrics.adoc

```diff
@@ -0,0 +1,44 @@
+// file included in the following:
+//
+// assembly-operator-monitoring.adoc
+
+[id='con-operator-scrape-metrics-{context}']
+= Scraping metrics
```

Comment:
I wonder if we should split this into two concepts, so that the proxy guide doesn't talk about operator metrics.